src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -117,6 +150,8 @@ private void initPool(HostAndPort master) {
         if (factory == null) {
           factory = new JedisFactory(master.getHost(), master.getPort(), connectionTimeout,
```

Comment:
It seems like you forgot to delete this row

Reply:
fixed https://github.com/xetorthio/jedis/pull/2024/commits/f232be700ec54a6b4cdf0d258b06b5db1b821c30